src/index.ts

```diff
@@ -10,6 +10,8 @@ import { fhirConfig, genericResources } from './config';
 const serverlessHandler = serverless(generateServerlessRouter(fhirConfig, genericResources), {
     request(request: any, event: any) {
         request.user = event.user;
+        request.tenantId = event.pathParameters.tenantId ? `-${event.pathParameters.tenantId}` : '';

```

Comment:
nit: I'd probably advice keeping the tenetId without the `-` and add the `-` when you need to
```suggestion
        request.tenantId = event.pathParameters.tenantId ? `${event.pathParameters.tenantId}` : '';
```